### PR TITLE
Create emojify.rb

### PR DESCRIPTION
**Pull request by Haroenv**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download emojify` is error-free.
- [x] `brew cask style --fix emojify` left no offenses.
- [x] `brew cask install emojify` worked successfully.
- [x] `brew cask uninstall emojify` worked successfully.

I'm not 100% sure that the link is correct, although it seems to work for me

### CODE REVIEW

File: Casks/emojify.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'emojify' do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/Haroenv/emojify-service/archive/#{version}.tar.gz"
```
Comment (vitorgalvao) [inline Casks/emojify.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by vitorgalvao** [comment]
Why the missing commands?

**Comment by Haroenv** [comment]
What do you mean?

File: Casks/emojify.rb (hunk 1 above)
Comment (Haroenv) [inline Casks/emojify.rb]: done

**Comment by vitorgalvao** [comment]
> What do you mean?

image

Why didn’t you run those?

**Comment by Haroenv** [comment]
@vitorgalvao the reason I didn't check those commands is because I'm getting this as error, while manually using curl with the same url works:


```
==> Downloading https://github.com/Haroenv/emojify-service/archive/latest.tar.gz
######################################################################## 100.0%
curl: (22) The requested URL returned error: 404 Not Found
Error: Download failed on Cask 'emojify' with message: Download failed: https://github.com/Haroenv/emojify-service/archive/latest.tar.gz
The incomplete download is cached at /Users/haroenv/Library/Caches/Homebrew/emojify-latest.tar.gz
```

I notice that getting the file via the browser works, but not via curl

File: Casks/emojify.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/emojify.rb]: Stanza is wrong (repeated).

**Comment by vitorgalvao** [comment]
You’re just downloading the source, anyway. Use the link in the releases page.

**Comment by Haroenv** [comment]
this `https://github.com/Haroenv/emojify-service/archive/v0.1.0.zip` ? So updating with every version then?

**Comment by vitorgalvao** [comment]
Yes, we prefer versioned URLs anyway.

File: Casks/emojify.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'emojify' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/emojify.rb]: You need to add the version and sha256.

**Comment by Haroenv** [comment]
Okay, I'll test that and change it, I run the `sha` on the downloaded file right?

**Comment by vitorgalvao** [comment]
> Okay, I'll test that and change it, I run the `sha` on the downloaded file right?

Yes. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/emojify.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'emojify' do
+  version '0.1.0'
+  sha256 '2ca93bf5707dfa67e4be8f1f27384ac1cb0d2644154999c7904b5d8b303ba18c'
+
+  url "https://github.com/Haroenv/emojify-service/archive/v#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/emojify.rb]: You’re still using the archive link, rather than the release.

Comment (Haroenv) [inline Casks/emojify.rb]: Oh, I used the link that came up in the github interface, I'll look for how to do it with a release link

Comment (vitorgalvao) [inline Casks/emojify.rb]: `https://github.com/Haroenv/emojify-service/archive/v0.1.0.zip` from the [releases page](https://github.com/Haroenv/emojify-service/releases).

Comment (Haroenv) [inline Casks/emojify.rb]: What am I missing, that's the same link? `"https://github.com/Haroenv/emojify-service/releases/download/v#{version}.zip"` maybe?

Comment (vitorgalvao) [inline Casks/emojify.rb]: Oh, right, it is. Because it’s a source download. Nevermind.

**Comment by Haroenv** [comment]
Thanks for the help you've given me 😄

**Comment by vitorgalvao** [comment]
Thank you for the contribution.

**Comment by Haroenv** [comment]
Uhm, I get this error now and the folder isn't in `/usr/local/Caskroom`, do you know why that is? 

```
brew cask install emojify
==> Downloading https://github.com/Haroenv/emojify-service/archive/v0.1.0.zip
######################################################################## 100.0%
==> Verifying checksum for Cask emojify
Error: It seems the Service source is not there: '/usr/local/Caskroom/emojify/0.1.0/Emojify.workflow'
```

**Comment by vitorgalvao** [comment]
There’s a directory inside the zip. Should be fixed now.

**Comment by Haroenv** [comment]
Yay, thanks for your work at cask!

**Pull request merged**